content/en/docs/reference/node/kubelet-checkpoint-api.md

```diff
@@ -25,6 +25,87 @@ should create the checkpoint archive to be only accessible by the `root` user. I
 is still important to remember if the checkpoint archive is transferred to another
 system all memory pages will be readable by the owner of the checkpoint archive.
 
+## Security risks and mitigation strategies
```

Comment:
it will be really nice to make this page a little more actionable. Maybe even convert to the task under the https://kubernetes.io/docs/tasks/administer-cluster/ and have an example instruction on how exactly each item can be achieved.

Reply:
I am undecided about these changes. They definitely sound like good ideas, but are also not very specific. Not sure how to describe it better.